bench/README.md

```diff
@@ -18,40 +28,243 @@ pip3 install -r bench/requirements.txt
 
 ## Preparation
 
+* Copy accounts keys to folder `~/.dcl/keyring-test`:
```

Comment:
How can it be copied? Please provide a command. How the accounts can be initially generated?

Reply:
Done! ✅